v3/src/components/graph/v2-graph-exporter.test.ts

```diff
@@ -124,6 +126,25 @@ describe("V2GraphImporter", () => {
     const { v2Document } = loadCodapDocument("mammals-all-diet-legends.codap")
     const v2GraphTiles = v2Document.components.filter(c => c.type === "DG.GraphView")
     v2GraphTiles.forEach(v2GraphTile => {
+      // console.log("v2GraphTile", v2GraphTile.componentStorage.title || v2GraphTile.componentStorage.name)
+      const v3GraphTile = v2GraphImporter({
+        v2Component: v2GraphTile,
+        v2Document,
+        insertTile: mockInsertTile
+      })
+      // tests round-trip import/export of every graph component
+      const v2GraphTileOut = v2GraphExporter({ tile: v3GraphTile! })
+      const v2GraphTileStorage = removePropertiesRecursive(v2GraphTile.componentStorage, kIgnoreProps)
+      const v2GraphTileOutStorage = removePropertiesRecursive(v2GraphTileOut?.componentStorage, kIgnoreProps)
+      expect(v2GraphTileOutStorage).toEqual(v2GraphTileStorage)
+    })
+  })
+
+  it("exports graph components with simple adornments", () => {
+    const { v2Document } = loadCodapDocument("mammals-simple-adornments.codap")
+    const v2GraphTiles = v2Document.components.filter(c => c.type === "DG.GraphView")
+    v2GraphTiles.forEach(v2GraphTile => {
+      // console.log("v2GraphTile", v2GraphTile.componentStorage.title || v2GraphTile.componentStorage.name)
```

Comment:
remove

Reply:
As mentioned elsewhere, I'm leaving these in because they're necessary for development/debugging. Maybe I'll wrap them in a function call that takes a boolean parameter so as to cause less confusion.